hack/make/tests/e2e.mk

```diff
@@ -52,7 +52,7 @@ test/e2e/cloudnative/upgrade: manifests/crd/helm
 
 ## Runs Application Monitoring e2e test only
 test/e2e/applicationmonitoring: manifests/crd/helm
-	go test -v -tags "$(shell ./hack/build/create_go_build_tags.sh true)" -count=1 ./test/scenarios/applicationmonitoring $(SKIPCLEANUP)
+	go test -v -tags "$(shell ./hack/build/create_go_build_tags.sh true)" -timeout 20m -count=1 ./test/scenarios/applicationmonitoring $(SKIPCLEANUP)
```

Comment:
same as in https://github.com/Dynatrace/dynatrace-operator/pull/1723#discussion_r1192016092 😉

Reply:
should rebase on `main` so you don't overwrite it